bitcoin/src/amount.rs

```diff
@@ -1544,6 +1547,30 @@ mod tests {
     use serde_test;
 
     use super::*;
+    
```

Comment:
There is trailing whitespace on this line. You can use the githooks to catch these (check out the README) for how to do that. If you are wondering how I knew, I have my shell and editor configured to highlight with red if there is trailing whitespace. I'll ack anyways, its no biggy - just something to know for next time. Cheers.